### PR TITLE
Feature/kah 95

### PR DESCRIPTION
**Pull request by ChBernat**

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
   },
   "husky": {
     "hooks": {
-      "pre-push": "npm run eslint && npm t"
+      "pre-commit": "npm run eslint && git add .",
```
Comment (theflexbird) [inline server/package.json]: remove this line.

File: server/env/config.test.json [hunk 2]
```diff
@@ -1 +1 @@
-{"upload_path":"uploads","interval_time":5000,"retries":3,"page_size":15,"serverKey":"48987fd129e56a7b4b6e412e87028eb0","dbURI":"mongodb://127.0.0.1:58303/bf06b852-1354-442f-af2f-9f5d1d62e8ec"}
\ No newline at end of file
+{"upload_path":"uploads","interval_time":5000,"retries":3,"page_size":15,"dbURI":"mongodb://127.0.0.1:64087/7e907d00-41a5-4e0d-889a-ce9679a8d151","serverKey":"9c4af8339b0979503823ed3e397c8f8b"}
```
Comment (theflexbird) [inline server/env/config.test.json]: It should not be in repo.

Comment (ChBernat) [reply server/env/config.test.json]: Mkay then I have to rework some serious part of the generation process of this config - should that be a part of this branch tho?

Comment (ChBernat) [reply server/env/config.test.json]: This will be resolved in a different branch.

File: server/api/services/scheduled-job.service.js [hunk 3]
```diff
@@ -1,97 +1,110 @@
...
-    if (!jobs[jobId]) {
-      return;
-    }
-    jobs[jobId].cancel();
-    delete jobs[jobId];
-  },
+  removeScheduledJob(jobId) {
+    const scheduledJob = ScheduledJob.findById(jobId);
+    const job = new Job(scheduledJob);
+    job.cancel();
```
Comment (theflexbird) [inline server/api/services/scheduled-job.service.js]: Should it not be also removed from DB

**Pull request closed without merging**